projects/fsastorefrontlib/src/cms-components/my-account/update-profile/update-profile.module.ts

```diff
@@ -29,7 +29,7 @@ import { FSUpdateProfileComponent } from './update-profile.component';
     FormErrorsModule,
   ],
   declarations: [FSUpdateProfileComponent, FSUpdateProfileFormComponent],
-  exports: [FSUpdateProfileComponent],
+  exports: [FSUpdateProfileComponent, FSUpdateProfileFormComponent],
   entryComponents: [FSUpdateProfileComponent],
```

Comment:
While we are at it, can we maybe remove `entryComponents`? They are no longer needed since Angular 9. And since we list Angular 10 as a requirement in installation instructions for FSA 2.0 and 2.1, there shouldn't be any problems for end user.